### PR TITLE
feat: expose server utils in `#content/server`

### LINKED ISSUE #1068: Expose nitro alias for runtime server utils
The idea is to expose runtime server utils under specific alias `#content/server`. Server composables should be exposed under this alias.

### PR DESCRIPTION
**Pull request by farnabaz**
### 🔗 Linked issue
resolve #1068

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Screen Shot 2022-05-12 at 1 45 21 AM

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/module.ts [hunk 1]
```diff
@@ -297,8 +293,25 @@ export default defineNuxtModule<ModuleOptions>({
...
       global: true
     })
-    // Register user global components
 
+    addTemplate({
+      filename: 'types/content.d.ts',
+      getContents: () => [
+        'declare module \'#content/server\' {',
+        `  import { QueryBuilder, ParsedContent, QueryBuilderParams } from "${relative(nuxt.options.buildDir, resolve('./runtime/types'))}";`,
+        '  const queryContent: <T = ParsedContent>() => QueryBuilder<T>;',
```
Comment (Atinux) [inline src/module.ts:302]: Shouldn't it be `serverQueryContent`?

Comment (farnabaz) [reply src/module.ts]: My bad, just pushed a patch

File: src/module.ts [hunk 2]
```diff
@@ -297,8 +293,25 @@ export default defineNuxtModule<ModuleOptions>({
...
       global: true
     })
-    // Register user global components
 
+    addTemplate({
+      filename: 'types/content.d.ts',
+      getContents: () => [
+        'declare module \'#content/server\' {',
+        `  import { QueryBuilder, ParsedContent, QueryBuilderParams } from "${relative(nuxt.options.buildDir, resolve('./runtime/types'))}";`,
+        '  const serverQueryContent: <T = ParsedContent>() => QueryBuilder<T>;',
```
Comment (Atinux) [inline src/module.ts:302]: I think the types are wrong, see https://github.com/nuxt/content/blob/feat/nitro-alias/src/runtime/server/storage.ts#L90

Comment (farnabaz) [reply src/module.ts:302]: Yes, I must mistakenly pushed my tests, before the `serverQueryContent` PR.

**Pull request merged**